### PR TITLE
Refactor/pagination a11y

### PR DESCRIPTION
**Pull request by ChrisPaj**
- @media screen and (max-width: 639px): width 100%
- no doubled button borders
- introduce prop 'small'
- button:focus with visible outlines
- adapt font size to 12px
- introduce button:active state

### CODE REVIEW

File: packages/components/src/html/index.html [hunk 1]
```diff
@@ -9,29 +9,1427 @@
     <title>Stencil Component Starter</title>
```
Comment (acstll) [inline packages/components/src/html/index.html:9]: This is probably a mistake. Please let's leave this `index.html` file as it is in `main` 🤓

File: packages/storybook-vue/stories/3_components/pagination/Pagination.stories.mdx [hunk 2]
```diff
@@ -2,117 +2,180 @@ import { Meta, Story, ArgsTable, Canvas } from '@storybook/addon-docs/blocks';
...
-    }}
+  title="Beta Components/Pagination"
+  component={ScalePagination}
+  argTypes={{
+    hideBorders: {
+      table: {
+        type: { summary: 'Boolean' },
+        defaultValue: { summary: false },
+      },
+      description: `(optional) Set to true to hide top and bottom borders`,
```
Comment (acstll) [inline packages/storybook-vue/stories/3_components/pagination/Pagination.stories.mdx:13]: Thank you for taking the time for adding these descriptions 🙏

**Review by acstll — CHANGES_REQUESTED** [review]
Thanks @ChrisPaj

I left some comments :nerd_face:

Please make sure all checks pass (prettier is failing because of the index.html, but you can leave that one alone).

Also, we don't want to merge this in `main` yet, we need the a11y approval first. So we keep the branch, and keep it up-to-date with `main` so it's easier to merge when the time comes.

File: packages/components/src/components/pagination/pagination.css [hunk 3]
```diff
@@ -11,31 +11,54 @@
...
+  --box-shadow-focus: inset 0 0 0 2px var(--scl-color-focus);
   --radius-first-prompt: 12px 0 0 12px;
   --radius-last-prompt: 0 12px 12px 0;
+  --radius-first-prompt-stack: 0 0 0 12px;
+  --radius-last-prompt-stack: 0 0 12px 0;
   --stroke-svg: var(--scl-color-black);
+  --stroke-svg-high-contrast: var(--scl-color-white);
+  --button-width: 42px;
+  --button-width-stack: 48px;
+  --info-padding: var(--scl-spacing-8);
```
[the remark this answers is not in the document]
Comment (ChrisPaj) [reply packages/components/src/components/pagination/pagination.css]: renamed some of the variables...

File: packages/components/src/components/pagination/pagination.css [hunk 4]
```diff
@@ -11,31 +11,54 @@
...
   --color-hover: var(--scl-color-primary-hover);
+  --color-active: var(--scl-color-primary-active);
   --color-button: var(--scl-color-grey-10);
   --border-button: var(--border);
-  --box-shadow-focus: 0px 0px 0px 2px var(--scl-color-focus);
+  --box-shadow-focus: inset 0 0 0 2px var(--scl-color-focus);
   --radius-first-prompt: 12px 0 0 12px;
   --radius-last-prompt: 0 12px 12px 0;
+  --radius-first-prompt-stack: 0 0 0 12px;
+  --radius-last-prompt-stack: 0 0 12px 0;
```
[the remark this answers is not in the document]
Comment (ChrisPaj) [reply packages/components/src/components/pagination/pagination.css]: I have introduced a new --radius variable

File: packages/components/src/components/pagination/pagination.css [hunk 5]
```diff
@@ -96,3 +134,46 @@ button:not(:disabled):hover svg {
   border-width: 1px !important;
   border-color: var(--color-hover) !important;
```
[the remark this answers is not in the document]
Comment (ChrisPaj) [reply packages/components/src/components/pagination/pagination.css]: yeah, looks like you can omit this...

File: packages/components/src/components/pagination/pagination.css [hunk 6]
```diff
@@ -96,3 +134,46 @@ button:not(:disabled):hover svg {
...
+    flex-direction: column;
+  }
+  .pagination__info-responsive {
+    display: initial;
+    order: 0;
+    overflow: auto;
+    border-left: var(--border);
+    border-right: var(--border);
+    border-bottom: 0;
+    border-radius: 12px 12px 0 0;
```
[the remark this answers is not in the document]
Comment (ChrisPaj) [reply packages/components/src/components/pagination/pagination.css]: ...use newly created radius variable for this now...

File: packages/components/src/components/pagination/pagination.tsx [hunk 7]
```diff
@@ -134,100 +136,108 @@ export class Pagination {
...
+                {start}-{end}
+              </span>{' '}
+              / {total}
+            </div>
+            <button
+              class={`${name}__first-prompt`}
+              part="first-prompt"
+              disabled={isAtStart}
+              onClick={() => this.goFirstPage()}
+              aria-label="Go to first page"
```
[the remark this answers is not in the document]
Comment (ChrisPaj) [reply packages/components/src/components/pagination/pagination.tsx]: I have created some new props for translation...

File: packages/components/src/html/index.html (hunk 1 above)
Comment (ChrisPaj) [reply packages/components/src/html/index.html]: That wasn't meant to be... I restored the original index.html

File: packages/storybook-vue/stories/3_components/pagination/Pagination.stories.mdx (hunk 2 above)
Comment (ChrisPaj) [reply packages/storybook-vue/stories/3_components/pagination/Pagination.stories.mdx:13]: ...like this one, too :-)

**Review by acstll — CHANGES_REQUESTED** [review]
Thank you @ChrisPaj, one more small thing 🙏

**Comment by acstll** [comment]
@ChrisPaj the `prettier` check is not passing… it complaints about the index.html file, maybe you can try running `yarn format` once more?

File: packages/storybook-vue/stories/3_components/pagination/Pagination.stories.mdx [hunk 8]
```diff
@@ -179,3 +183,35 @@ export const TemplateEmbedded = (args, { argTypes }) => ({
  <scale-pagination page-size="5" start-element="100" total-elements="200" hide-borders></scale-pagination>
 </div>
 ```
+
+## Translation German
```
[the remark this answers is not in the document]
Comment (ChrisPaj) [reply packages/storybook-vue/stories/3_components/pagination/Pagination.stories.mdx]: ok, changed...

File: packages/components/src/components/pagination/pagination.tsx [hunk 9]
```diff
@@ -58,7 +58,15 @@ export class Pagination {
...
-  @Prop() small = false;
+  @Prop() small = true;
+  /** (optional) translation to 'Go to first page'  */
+  @Prop() langGoFirst = 'Go to first page';
+  /** (optional) translation to 'Go to next page'  */
+  @Prop() langGoNextt = 'Go to next page';
+  /** (optional) translation to 'Go to previous page'  */
+  @Prop() langGoPrevious = 'Go to previous page';
+  /** (optional) translation to 'Go to last page'  */
+  @Prop() langGoLast = 'Go to last page';
```
[the remark this answers is not in the document]
Comment (ChrisPaj) [reply packages/components/src/components/pagination/pagination.tsx]: props have been renamed...

**Review by acstll — APPROVED** [review]

**Review by AnnaKuchtin — CHANGES_REQUESTED** [review]
VirtualBox_MSEdge - Win10_12_08_2021_11_01_40
In HCM the icons are not being displayed anymore 🙈

**Comment by timheiler-ico** [comment]
we are not using scale icons for next / prev?
Icon / Navigation / Right
Icon / Navigation / Left
would be great?

**Comment by timheiler-ico** [comment]
[ ] icons on mobile: 16x16 px (like desktop)
[ ] total mobile navigation height: 80px (40px top, 40px bottom part)

**Review by AnnaKuchtin — APPROVED** [review]

**Comment by acstll** [comment]
> we are not using scale icons for next / prev?
> Icon / Navigation / Right
> Icon / Navigation / Left
> would be great?

I left them all inlined because they behave differently in HCM, would it be possible to add the other "double arrow" icons added to the library?

I did reduce the size of the inlined svgs to match the `16` size of the `scale-icon` icons…

**Comment by acstll** [comment]
We're sending it out and closing for now until we get more feedback.

(The visual tests need to be updated!)

**Pull request closed without merging**